.github/workflows/danger_pr.yml

```diff
@@ -0,0 +1,16 @@
+name: Run Danger on PR
+
+on:
+  pull_request:
+    types: [opened, edited, reopened]
```

Comment:
Given we don't do any code related check, this doesn't re run when you push a new commit